src/Content/ContentManager.py

```diff
@@ -677,7 +677,8 @@ def sign(self, inner_path="content.json", privatekey=None, filewrite=True, updat
                 content = {"files": {}, "signs": {}}  # Default content.json
 
             if inner_path == "content.json":  # It's the root content.json, add some more fields
-                content["title"] = "%s - ZeroNet_" % self.site.address
+                content["title"] = "%s" % self.site.address
+                content["optional"] = "(video/.*|audio/.*)"
```

Comment:
Is this change necessary? It looks like a backward compatibility breaker and also I really don't understand why specific directory names should be hardcoded.

Reply:
No, is not necessary.

Reply:
Can you remove this line then please?

Reply:
Done.